proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -139,8 +142,26 @@ message SearchCommitReferenceRequest {
   uint32 page_token = 5;
 }
 
-message SearchCommitReferenceResponse {
-  repeated RepositoryCommit repository_commits = 1;
+message SearchTagResponse {
+  repeated RepositoryTag repository_tags = 1;
+  // There are no more pages if this is 0.
+  uint32 next_page_token = 2;
+}
+
+message SearchDraftRequest {
+  // The owner of the repository.
+  string repository_owner = 1;
+  // The name of the repository.
+  string repository_name = 2;
+  // The search string.
+  string query = 3;
+  uint32 page_size = 4;
+  // The first page is returned if this is 0.
+  uint32 page_token = 5;
+}
+
+message SearchDraftResponse {
+  repeated RepositoryDraft repository_drafts = 1;
```

Comment:
Can we make this return repository draft commits instead to keep it consistent with the [`ListRepositoryDraftCommits`](https://buf.build/bufbuild/buf/docs/main:buf.alpha.registry.v1alpha1#buf.alpha.registry.v1alpha1.RepositoryCommitService.ListRepositoryDraftCommits)?